ui/tests/email.test.ts

```diff
@@ -0,0 +1,53 @@
+import { DataLayer } from '@Utils/dataLayer';
+import { test, expect } from '@playwright/test';
+
+test.describe('testing case when we are fill email success', () => {
```

Comment:
better to use description smth like this: 'check event in data layer after subscription'